### DISCUSSION TITLE
Web DJ Standalone

### DISCUSSION THREAD

**Discussion by renggap** [AzuraCast/AzuraCast#5087] [General]
Hello, 

The Web DJ feature is amazing. Is there any standalone package so I can install it on a server, which will broadcast to any SHOUTcast server? 
Some folks may need this to broadcast from Chromebook.

**Comment by BusterNeece** [comment]
@renggap Our Web DJ feature is built upon https://webcast.github.io/ which works with Liquidsoap directly, so any Liquidsoap server can use it; we just auto-configure our setup to use our own DJ credentials and to input all of the necessary details automatically.

**Reply by renggap** [reply]
Thank you! Will look into it.
